File: 5.c
#include <stdio.h>

int main() 
{
    double marks;
    printf("Enter the marks: ");
    scanf("%lf", &marks);
    if (marks >= 40) 
    {
        printf("Passed\n");
    }
    else 
    {
        printf("Failed\n");
    }
    return 0;
}

File: 5.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
rl.question("Enter the marks: ", function(input) {
    const marks = parseFloat(input);
    if (marks >= 40) {
        console.log("Passed");
    } else {
        console.log("Failed");
    }
    rl.close();
});
